scripts/prepare-gh-pages-deployment.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# This apply needed changes before running yarn build,
+# as described in https://github.com/FLOIP/flow-builder/blob/master/gh-pages-deploy.md
+
+echo "Prepare gh-pages deployment"
+git apply patches/gh-pages-prepare-router.patch
```

Comment:
@safydy-r this is brilliant + elegant to use git patches! 

What happens in the future as we make changes to the affected sources? Will we quite quickly end up with conflicts, or will git handle that intelligently?

Reply:
Thanks 😊 .
In future, in case on conflict, we will correct the patch file. Let me add some notes in this file